server/src/utils/media.ts

```diff
@@ -394,6 +394,33 @@ export class VP9Config extends BaseConfig {
   }
 }
 
+export class AV1Config extends BaseConfig {
+  getPresetOptions() {
+    const speed = this.getPresetIndex() + 4; // Use 4 as slowest, giving us an effective range of 4-12 which is far more useful than 0-8
+    if (speed >= 0) {
+      return [`-preset ${speed}`];
+    }
+    return [];
+  }
+
+  getBitrateOptions() {
+    const bitrates = this.getBitrateDistribution();
+    if (bitrates.max > 0) {
+      return [
+        `-crf ${this.config.crf}`,
+        `-svtav1-params mbr=${bitrates.max}${bitrates.unit}`, // Wrapping the params in quotes breaks them for some reason, so don't do that
+      ];
+    } else {
+      return [`-crf ${this.config.crf}`];
+    }
+  }
+
+  getThreadOptions() {
+    return [];
```

Comment:
You should only add this if `threads` is a positive number.

Reply:
Is this for readability?

Because SVT will default to using all threads anyway if 0 or a neg number is passed.
I can add a check for it anyway.

Reply:
Yeah, it's just easier to know for a fact that it works a certain way than to leave it ambiguous. e.g. 0 is optimal, but is -1 the same as 0? Does it accept -2? Can there be a difference between passing `-threads 0` and omitting it? Do all the encoders have this same behavior?